data/strings/strings.txt

```diff
@@ -10870,6 +11111,7 @@
     ja = マップをダウンロードし、複数マップを利用してより最適なルートを作成しますか？
     ko = 지도를 다운로드하여, 두 개 이상의 지도를 통과하는 더 최적화된 경로를 설정하시겠습니까?
     lt = Norėtumėte atsisiųsti žemėlapį, kad sukurtumėte labiau optimalų maršrutą, apimantį daugiau nei vieną žemėlapį?
+    lv = Vai vēlaties lejupielādēt karti un izvaidot optimālāku maršrutu, kas ietvar vairāk par vienu karti?
```

Comment:
izvaidot -> izveidot
ietvar -> ietver

Reply:
"Optimālāku" is an incorrect form - suggesting "piemērotāku" ("optimāls" already is the highest possible level, cannot gain "-āks").

Reply:
Yes, so with both corrections — "Vai vēlaties lejupielādēt karti un izveidot piemērotāku maršrutu, kas ietvar vairāk par vienu karti?"

Reply:
You made this mistake again:
ietvar -> ietver

Reply:
Sorry, tired brain, it should be correct now — "Vai vēlaties lejupielādēt karti un izveidot piemērotāku maršrutu, kas ietver vairāk par vienu karti?"